### PR TITLE
Install default version of nodejs

### PR DESCRIPTION
**Pull request by jungziege**

### CODE REVIEW

File: targets/ubuntu-14.04-enterprise/Dockerfile [hunk 1]
```diff
@@ -115,3 +97,7 @@ RUN echo $IMAGE_TAG > /opt/circleci/image_version
 ADD targets/ubuntu-14.04-enterprise/pkg-versions.sh /opt/circleci/bin/pkg-versions.sh
 
 LABEL circleci.user="ubuntu"
+
+USER ubuntu
+
+CMD ["/bin/bash"]
```
Comment (kimh) [inline targets/ubuntu-14.04-enterprise/Dockerfile]: I assume this is for when you want to run this container locally with `docker run` command? I used to do something like `docker run -it circleci/build-image:ubuntu-14.04-XXL-913-a78740d su - ubuntu` but I think `CMD` is better.

Comment (jungziege) [inline targets/ubuntu-14.04-enterprise/Dockerfile]: I just copied it from the .com container.

Comment (kimh) [inline targets/ubuntu-14.04-enterprise/Dockerfile]: I see. The XL image is for picard only where you don't need `sbin/init` but it's required in 1.0. I wasn't fully sure if adding `CMD` breaks but sounds like it did as George mentioned.

Comment (jungziege) [inline targets/ubuntu-14.04-enterprise/Dockerfile]: Ah, so I should not have added the CMD line? Where did George mention that this broke things?

**Pull request merged**